src/main/java/org/broadinstitute/hellbender/tools/spark/transforms/markduplicates/MarkDuplicatesSparkUtils.java

```diff
@@ -474,38 +480,43 @@ public static void saveMetricsRDD(final MetricsFile<GATKDuplicationMetrics, Doub
     }
 
     /**
-     * Comparator for sorting Reads by coordinate. Note that a header is required in
-     * order to meaningfully compare contigs.
+     * Comparator for by their PhysicalLoccation attributes and strandedness. This comparator is intended to serve as a tiebreaker
+     * for the score comparator.
      *
-     * Uses the various other fields in a read to break ties for reads that share
-     * the same location.
+     * NOTE: we don't need to worry about reads genomic location as they will necessarily be grouped by start positions if they are being compared
      *
-     * Ordering is not the same as {@link htsjdk.samtools.SAMRecordCoordinateComparator}.
-     * It compares two pairedEnds objects by their first reads according to first their clipped start positions
-     * (they were matched together based on UnclippedStartOriginally), then the orientation of the strand, followed by
-     * the readname lexicographical order.
+     * Ordering is not the same as {@link htsjdk.samtools.SAMRecordCoordinateComparator}, except for the alignment position.
```

Comment:
this comment is still out of date, it doesn't look at the alignment position anymore